src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,35 @@
+package duke;
+import java.io.*;
+import java.util.*;
+
+import duke.tasks.*;
+import duke.tasklist.*;
+import duke.UI.*;
+
+public class Duke {
+
+    public static void main(String[] args) throws IOException {
+        UI.printGreeting();
+        UI.printCommands();
+        Scanner sc = new Scanner(System.in);
+        String currInput = null;
+        Pikachu pikachu = new Pikachu();
+        Storage storage = new Storage("Tasklist.txt", pikachu);
+        storage.readTaskList();
+
+        //Start accepting commands
+        while (true) {
+            currInput = sc.nextLine(); //scan in user input
+            if (currInput.toLowerCase().equals("bye")) break; //if user input == bye, exit programme
```

Comment:
I believe 1-liner if clauses do not comply with the coding standard. Also, shouldn't there be curly braces for one-liner if statements?
```suggestion
            if (currInput.toLowerCase().equals("bye")) {
                break; //if user input == bye, exit programme
            }
```